### ISSUE TITLE
Font Names with spaces not correctly escaped

### ISSUE THREAD

**Issue by nicolas-schreiber** [wkhtmltopdf/wkhtmltopdf#3857]
Hi,
I have a problem with embedded fonts with spaces in their name.
In my case this is with Source Han Sans.
This is the desired output with SourceHanSans embedded:
```
<< /Type /Font
/Subtype /CIDFontType2
/BaseFont /Source#20Han#20Sans#20Bold
/CIDSystemInfo << /Registry (Adobe) /Ordering (Identity) /Supplement 0 >>
/FontDescriptor 13 0 R
/CIDToGIDMap /Identity
/DW 992 >>
```

But wkhtmltopdf creates following:
```
<< /Type /Font
/Subtype /CIDFontType2
/BaseFont /Source Han Sans Bold
/CIDSystemInfo << /Registry (Adobe) /Ordering (Identity) /Supplement 0 >>
/FontDescriptor 13 0 R
/CIDToGIDMap /Identity
/DW 992 >>
```

The poppler-utils pdffonts spits out a lot of `Dictionary key must be a name object` errors with the current output and the PDFViewers show blank screens.

If I manually escape that /BaseFont it works like a charm.

**Comment by nicolas-schreiber** [comment]
And I just found a duplicate, sorry: https://github.com/wkhtmltopdf/wkhtmltopdf/issues/3672
But I hope this is some good additional information.

**Comment by ashkulz** [comment]
A release candidate [is available](https://wkhtmltopdf.org/downloads.html#release-candidate) which includes the fixes made in PR wkhtmltopdf/qt#38 -- please test and report your findings before the final release.
